Content.Server/Administration/Commands/BriefCommand.cs

```diff
@@ -0,0 +1,142 @@
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Players;
+using Content.Shared.Administration;
+using Content.Shared.Roles;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Administration.Commands.Brief
+{
+    [AdminCommand(AdminFlags.Admin)]
+    public sealed class BriefCommand : IConsoleCommand
+    {
+        [Dependency] private readonly IEntityManager _entities = default!;
+        [Dependency] private readonly IEntitySystemManager _entitysys = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+        public string Command => "brief";
+        public string Description => "Makes you a mob of choice until the command is rerun.";
+        public string Help => $"Usage: {Command} <outfit> <name> <entity> <force>";
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            var player = shell.Player as IPlayerSession;
+            if (player == null)
+            {
+                shell.WriteLine("You aren't a player.");
+                return;
+            }
+
+            var mind = player.ContentData()?.Mind;
+
+            if (mind == null)
+            {
+                shell.WriteLine("You can't spawn without a mind.");
+                return;
+            }
+
+            if (mind.VisitingEntity != default)
+            {
+                var didYouBrief = false;
+                if (args.Length >= 4)
+                    if (args[3].ToLower() == "true")
+                        didYouBrief = true;
+
+                foreach (var officer in _entities.EntityQuery<BriefOfficerComponent>(true))
+                {
+                    if (officer.Owner == mind.VisitingEntity)
+                    {
+                        didYouBrief = true;
+                        player.ContentData()!.Mind?.UnVisit();
+                        _entities.QueueDeleteEntity(officer.Owner);
+                        return;
+                    }
+                }
+
+                if (didYouBrief == false)
+                {
+                    shell.WriteError("You are visiting something other than a brief already.");
+                    return;
+                }
+            }
+
+            var outfit = "CentcomGear";
+            if (args.Length >= 1)
+                if (_prototypeManager.TryIndex<StartingGearPrototype>(args[0], out var outfitProto))
+                    outfit = outfitProto.ID;
+                else
+                {
+                    shell.WriteError("Given outfit is invalid.");
+                    return;
+                }
```

Comment:
This is a braceless if longer than two lines. Add braces. Additionally you mixed a braceless if with a braced else, once again add braces.